clients/js/test/add.collections.test.ts

```diff
@@ -83,6 +86,34 @@ if (!process.env.COHERE_API_KEY) {
   });
 }
 
+if (!process.env.AWS_ACCESS_KEY_ID || !process.env.AWS_SECRET_ACCESS_KEY) {
```

Comment:
Do you mind moving this to a separate file under `test/embeddings/aws.bedrock.test.ts`? This file is getting very crowded.

Reply:
Splitted https://github.com/chroma-core/chroma/pull/1659/commits/5a6b8ec9d37295e92dd3c45295854d56ac64aede